### PR TITLE
Add StoreSections component.

### PR DESCRIPTION
**Pull request by justincan**
- Copied over from snapweb (StoreHeader).

### CODE REVIEW

File: src/Store/StoreSections.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
```
Comment (earnubs) [inline src/Store/StoreSections.js]: Hi! You can also use import to grab module members, which makes the code a little neater, like this:
```
import React, { Component, PropTypes } from "react";
...
export default class StoreSections extends Component {
  static propTypes = {
    title: PropTypes.string
  }
}
```

File: src/index.js [hunk 2]
```diff
@@ -21,3 +21,4 @@ export { default as SnapPageLogin } from './SnapPage/SnapPageLogin'
 export { default as SnapPagePurchase } from './SnapPage/SnapPagePurchase'
 export { default as SnapPageSummary } from './SnapPage/SnapPageSummary'
 export { default as SnapPageTags } from './SnapPage/SnapPageTags'
+export { default as StoreSections } from './Store/StoreSections'
```
Comment (earnubs) [inline src/index.js:26]: I think these can just be:
```
export StoreSections from './Store/StoreSections';
```

but maybe there's a reason for doing it this way?

Comment (justincan) [reply src/index.js:26]: That seems fine, it's exporting the default export.

File: src/Store/StoreSections.js (hunk 1 above)
Comment (justincan) [reply src/Store/StoreSections.js]: I'm taking this as a comment - I'd rather leave it. It has all the usual (minimal) advantages and (larger) disadvantages of global importation. It's not a common idiom either.

Comment (earnubs) [reply src/Store/StoreSections.js]: Hi, can you explain a little more what you mean? What are the disadvantages? Which bit isn't common?

**Pull request merged**